app/backend/app.py

```diff
@@ -95,13 +77,13 @@ def content_file(path):
     blob_file.seek(0)
     return send_file(blob_file, mimetype=mime_type, as_attachment=False, download_name=path)
 
-@app.route("/ask", methods=["POST"])
+@bp.route("/ask", methods=["POST"])
 def ask():
-    if not request.json:
-        return jsonify({"error": "request must be json"}), 400
+    if not request.is_json:
```

Comment:
This was a bug, caught by the new tests. Calling request.json causes an immediate 415 HTTP error if theres no JSON passed in, thus it's now request.is_json.